contrib/mod_exec.c

```diff
@@ -317,7 +317,10 @@ static void exec_prepare_fds(int stdin_fd, int stdout_fd, int stderr_fd) {
    * First, use getrlimit() to obtain the maximum number of open files
    * for this process -- then close that number.
    */
-#if defined(RLIMIT_NOFILE) || defined(RLIMIT_OFILE)
+#if defined(__APPLE__)
```

Comment:
Rather than using a platform-specific macro like `__APPLE__`, I'd prefer to use either a) something feature-based, or b) something that the proftpd build system already provides/sets, e.g. `DARWIN9`, `DARWIN10`, `DARWIN11`, etc.

Also, is this change fixing a specific bug, or is it making something better?  It'd hard to tell from just the description.  I.e. should `RLIMIT_OFILE` be preferred over `RLIMIT_NOFILE`, but only on some platforms, or...?